### DISCUSSION TITLE
Batch export dashboards

### DISCUSSION THREAD

**Discussion by nabelekt** [grafana/grafana#64473] [Dashboards, Panels & Visualizations]
I know I created an issue for this before tickets were moved to Discussions, but can't find it.

We often need to export our dashboards. Doing them all one-by-one is tedious. The fact that [this tool](https://gist.github.com/crisidev/bd52bdcc7f029be2f295) exists is further proof that this functionality is desired.

Bonus points if we can specify the naming pattern for the exported json files. We'd rather not have the datetime stamp included in in the filenames.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82829. Feel free to continue the discussion around this there. Thank you!
